packages/atlas/pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: atlas
 description: An extensible map abstraction for Flutter with support for multiple map providers
-version: 0.12.1
+version: 0.13.0
```

```suggestion
version: 0.12.2
```

Since this isn't a breaking change, I think it should just be bumped to 0.12.2.